### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by bentonam**

### CODE REVIEW

File: tasks/preflight.yml [hunk 1]
```diff
@@ -51,14 +56,41 @@
...
+    - name: Couchbase Server Version
+      debug:
+        msg: "{{ couchbase_server_version }}"
+
+    - name: Fail when the installed version of Couchbase is 7.x or greater
+      fail:
+        msg: >
+          The CMOS Couchbase Exporter does not need to be installed when using Couchbase 7.x+ metrics are natively exposed through a /metrics endpoint.
+          See: https://docs.couchbase.com/server/current/manage/monitor/set-up-prometheus-for-monitoring.html
+      when: (couchbase_server_version | split('.') | first | int) >= 7
```
Comment (markspolakovs) [inline tasks/preflight.yml:84]: Could this be `couchbase_server_version is version('7.0.0', '>=')`? https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions

Comment (bentonam) [reply tasks/preflight.yml]: Yes that should work, updated.

File: tasks/preflight.yml [hunk 2]
```diff
@@ -51,14 +56,41 @@
...
+        url: http://{{ inventory_hostname }}:8091/pools/default
+        method: GET
+        user: "{{ couchbase_username }}"
+        password: "{{ couchbase_password }}"
+        timeout: 2
+      register: _results
+
+    - name: Set the Couchbase Server Version from the result of API call
+      set_fact:
+        couchbase_server_version: "{{ _results.json | community.general.json_query('nodes[?thisNode].version | [0]') }}"
```
Comment (markspolakovs) [inline tasks/preflight.yml:72]: Does `community.general` need to be added to the requirements file? 

>This collection is shipped with the Ansible package. So if you have it installed, no more action is required.

> If you have a minimal installation (only Ansible Core installed) or you want to use the latest version of the collection along with the whole Ansible package, you need to install the collection from [Ansible Galaxy](https://galaxy.ansible.com/community/general) manually with the ansible-galaxy command-line tool:

**Review by markspolakovs — APPROVED** [review]

File: tasks/preflight.yml (hunk 2 above)
Comment (bentonam) [reply tasks/preflight.yml:72]: Roles can't rely on external collections through requirements.  Might need to look at a different way of achieving the same thing as `json_query`

**Pull request merged**